CMakeModules/FindGROMACS.cmake

```diff
@@ -22,51 +22,52 @@
 
 find_library(GROMACS_LIBRARY NAMES gromacs_d gromacs HINTS ${PC_GROMACS_D_LIBRARY_DIRS} ${PC_GROMACS_LIBRARY_DIRS})
 if (GROMACS_LIBRARY)
-  include(CheckLibraryExists)
   include(CheckCXXLibraryExists)
-  check_library_exists("${GROMACS_LIBRARY}" GromacsVersion "" FOUND_GROMACS_VERSION)
-  if(NOT FOUND_GROMACS_VERSION)
-    check_cxx_library_exists("${GROMACS_LIBRARY}" gmx_version "" FOUND_GROMACS_VERSION_CXX)
-  endif()
-  if(NOT FOUND_GROMACS_VERSION AND NOT FOUND_GROMACS_VERSION_CXX)
-    message(FATAL_ERROR "Could not find GromacsVersion in ${GROMACS_LIBRARY}, take look at the error message in ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeError.log to find out what was going wrong. If you don't have pkg-config installed you will most likely have to set GROMACS_LIBRARY by hand which sets the gromacs lib and it's dependencies (i.e. -DGROMACS_LIBRARY='/path/to/libgmx.so;/path/to/libblas.so;/path/to/libm.so')!")
-  endif()
-  check_library_exists("${GROMACS_LIBRARY}" gmx_gpu_sharing_supported "" FOUND_GROMACS_GMX_GPU_SHARING_SUPPORTED)
-  #check is above
-  if(FOUND_GROMACS_VERSION_CXX)
-    set(GROMACS_VERSION 52)
-  elseif(FOUND_GROMACS_GMX_GPU_SHARING_SUPPORTED)
-    set(GROMACS_VERSION 51)
-  else()
-    message(FATAL_ERROR "Could not find gmx_version nor gmx_gpu_sharing_supported in the gromacs library, take look at the error message in ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeError.log to find out what was going wrong. This most likely means that your gromacs version is too old, we need at least gromacs 5.1 !") 
-  endif()
   check_cxx_library_exists("${GROMACS_LIBRARY}" gmx_is_single_precision "" FOUND_GMX_IS_SINGLE_PRECISION)
   check_cxx_library_exists("${GROMACS_LIBRARY}" gmx_is_double_precision "" FOUND_GMX_IS_DOUBLE_PRECISION)
-  if(FOUND_GMX_IS_DOUBLE_PRECISION AND GROMACS_VERSION GREATER 51)
+  if(FOUND_GMX_IS_DOUBLE_PRECISION)
     set(GROMACS_DEFINITIONS "-DGMX_DOUBLE=1")
-  elseif(FOUND_GMX_IS_SINGLE_PRECISION AND GROMACS_VERSION GREATER 51)
+  elseif(FOUND_GMX_IS_SINGLE_PRECISION)
     set(GROMACS_DEFINITIONS "-DGMX_DOUBLE=0")
-  elseif(FOUND_GMX_IS_DOUBLE_PRECISION)
-    set(GROMACS_DEFINITIONS "-DGMX_DOUBLE")
   elseif(NOT FOUND_GMX_IS_SINGLE_PRECISION)
-    message(FATAL_ERROR "Could not find neither gmx_is_single_precision nor gmx_is_double_precision in the gromacs library, that is very very strange, take look at the error message in ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeError.log to find out what was going wrong. This most likely means that your gromacs version is too old, we need at least gromacs 5.1 !") 
+    message(FATAL_ERROR "Could not find neither gmx_is_single_precision nor gmx_is_double_precision in the gromacs library, that is very very strange, take look at the error message in ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeError.log to find out what was going wrong. This most likely means that your gromacs version is too old, we need at least gromacs 2016 !") 
```

Comment:
Could not find. Neither gmx_is_single_precision nor gmx_is_double_precision in the gromacs library, that is very very strange, take a look at the error message in ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeError.log to find out what was going wrong. This most likely means that your gromacs version is too old, we need at least gromacs 2016!

Reply:
This is the only thing that I would change the English could use a little tweak in the error message.

Reply:
What text would you suggest?

Reply:
What I have copied in the first comment.